ConcernsCaseWork/Service.TRAMS/NtiWarningLetter/NtiWarningLetterDto.cs

```diff
@@ -8,13 +8,15 @@ public class NtiWarningLetterDto
 	{
 		public long Id { get; set; }
 		public long CaseUrn { get; set; }
-		public DateTime CreatedAt { get; set; }
-		public DateTime UpdatedAt { get; set; }
+		public DateTime? DateLetterSent { get; set; }
 		public string Notes { get; set; }
+		public int? StatusId { get; set; }
+		public ICollection<int> WarningLetterReasonsMapping { get; set; }
+		public ICollection<int> WarningLetterConditionsMapping { get; set; }
+		public string CreatedBy { get; set; }
+		public DateTime CreatedAt { get; set; }
+		public DateTime? UpdatedAt { get; set; }
 		public DateTime? ClosedAt { get; set; }
-		public NtiWarningLetterStatusDto Status { get; set; }
-		public ICollection<NtiWarningLetterReasonDto> Reasons { get; set; }
-		public DateTime? SentDate { get; set; }
-
+		public int? ClosedStatusId { get; set; }
```

Comment:
ClosedStatusId is never used (also I'm not sure what it would refer to - there is already a StatusId, but that is probably just me not having all the info)

Reply:
Closed status is a different state. This will be populated when a case is populated. A story following this will use the field.